Makefile

```diff
@@ -147,7 +147,8 @@ lb-csi-manifests: verify_image_registry deploy/k8s
 		--set kubeVersion=v1.17 \
 		--set imageRegistry=$(DOCKER_REGISTRY) \
 		--set sidecarImageRegistry=$(SIDECAR_DOCKER_REGISTRY) \
-		--set image=$(DOCKER_TAG) > deploy/k8s/lb-csi-plugin-k8s-v1.17-dc.yaml
+		--set image=$(DOCKER_TAG) \
+		--set discoveryClientImage=$(DISCOVERY_CLIENT_DOCKER_TAG) > deploy/k8s/lb-csi-plugin-k8s-v1.17-dc.yaml
```

Comment:
I think we're adding more maintenance complexity  to this already complex makefile. 
How about if you this ?
 
The actual change is minimum from one version to the other and follows a strict order. 
Move the subpart of **deploy/k8s/lb-csi-plugin-k8s-v1.28.yaml** and **deploy/k8s/lb-csi-plugin-k8s-v1.28-dc.yaml**    [ k8s-v1.17 - k8s-v1.30 ] into a separate input file as input of versionList.

Repeat loop for  **helm template** as many times you have entries in versionList. This will reduce all these repeated lines 133 - 385 to 7/8 lines. 


e.g 
helm template deploy/helm/lb-csi/ \
		--namespace=kube-system \
		--set allowExpandVolume=true \
		--set enableSnapshot=true \
		--set discoveryClientInContainer=true \
		--set kubeVersion=**version[i]** \
		--set imageRegistry=$(DOCKER_REGISTRY) \
		--set sidecarImageRegistry=$(SIDECAR_DOCKER_REGISTRY) \
		--set image=$(DOCKER_TAG) \
		--set discoveryClientImage=$(DISCOVERY_CLIENT_DOCKER_TAG) > deploy/k8s/lb-csi-plugin-**version[i]** -dc.yaml

Reply:
let me know your thoughts .........